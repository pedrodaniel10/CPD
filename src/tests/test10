8 5 1000000 50