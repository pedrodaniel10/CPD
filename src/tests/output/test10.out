0.26 0.83
0.50 0.50
